src/main/scala/com/ibm/couchdb/api/builders/GetManyDocumentsQueryBuilder.scala

```diff
@@ -43,87 +59,136 @@ case class GetManyDocumentsQueryBuilder(
           """.stripMargin)
   }
 
-  def conflicts(conflicts: Boolean = true): GetManyDocumentsQueryBuilder = {
+  def conflicts(
+      conflicts: Boolean = true): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
     set("conflicts", conflicts)
   }
 
-  def descending(descending: Boolean = true): GetManyDocumentsQueryBuilder = {
+  def descending(
+      descending: Boolean = true): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
     set("descending", descending)
   }
 
-  def endKey[K: W](endKey: K): GetManyDocumentsQueryBuilder = {
+  def endKey[K: W](endKey: K): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
     set("endkey", write(endKey))
   }
 
-  def endKeyDocId(endKeyDocId: String): GetManyDocumentsQueryBuilder = {
+  def endKeyDocId(
+      endKeyDocId: String): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
     set("endkey_docid", endKeyDocId)
   }
 
-  private def includeDocs(includeDocs: Boolean = true): GetManyDocumentsQueryBuilder = {
-    set("include_docs", includeDocs)
+  def includeDocs[D: R]: GetManyDocumentsQueryBuilder[IncludeDocs[D], AM, BT] = {
+    set("include_docs", true)
+  }
+
+  def excludeDocs: GetManyDocumentsQueryBuilder[ExcludeDocs, AM, BT] = {
+    set("include_docs", false)
+  }
+
+  def allowMissing: GetManyDocumentsQueryBuilder[ID, MissingAllowed, BT] = {
+    allowMissing(allow = true)
+  }
+
+  def allowMissing(allow: Boolean): GetManyDocumentsQueryBuilder[ID, MissingAllowed, BT] = {
```

Comment:
This doesn't look right, `allow` is unused, and the return type is always `MissingAllowed`.